cmd/di.go

```diff
@@ -160,7 +160,7 @@ type Dependencies struct {
 	ConsumerTotalsStorage    *pingpong.ConsumerTotalsStorage
 	AccountantPromiseStorage *pingpong.AccountantPromiseStorage
 	ConsumerBalanceTracker   *pingpong.ConsumerBalanceTracker
-	AccountantPromiseSettler *pingpong.AccountantPromiseSettler
+	AccountantPromiseSettler pingpong.AccountantPromiseSettlerInterface
```

Comment:
Can we remove the `Interface` word from the name? It's already clear that this is an interface, no need to duplicate it in the name. 
```suggestion
	AccountantPromiseSettler pingpong.AccountantPromiseSettler
```